ext/symengine/symengine.c

```diff
@@ -44,6 +45,9 @@ void Init_symengine() {
     rb_define_method(c_basic, "subs", cbasic_subs, -1);
     rb_define_method(c_basic, "coerce", cbasic_coerce, 1);
 
+    //Sympify as a Module Level Function
+    rb_define_module_function(m_symengine, "sympify", cutils_sympify, 1);
```

Comment:
I'm not sure that "sympify" is a good name for Ruby API at all. As far as I can understand, it is derived from "sympy" (old symengine name), which is not what we are working with :) (And, in any case, hard to remember/understand.)

What do you think about a more generic name, like `coerce` or `convert`?
And, speaking about short name, I think something like `SymEngine[expression]` or even `SymEngine(expression)` is the most reasonable.